front/tests/pages/operational-studies/scenario-timetable-section.ts

```diff
@@ -167,15 +243,54 @@ class ScenarioTimetableSection extends CommonPage {
     await expect(this.timetableTrains).toHaveCount(trainCount);
   }
 
+  // Verify that the total items label matches the expected syntax (plural only)
+  async verifyTotalItemsLabel(
+    translations: TimetableFilterTranslations & CommonTranslations,
+    itemCounts: {
+      totalPacedTrainCount: number;
+      totalTrainScheduleCount: number;
+    }
+  ): Promise<void> {
+    const { totalPacedTrainCount, totalTrainScheduleCount } = itemCounts;
+    await expect(this.timetableTotalItemLabel).toBeVisible();
+
+    // Total items label has the syntax : "X services and Y trains"
+    const pacedTrainLabel = translations.pacedTrain_other.split(' ')[1]; // "services"
+    const trainScheduleLabel = translations.train_other.split(' ')[1]; // "trains"
+    const expectedComputedLabel = `${totalPacedTrainCount} ${pacedTrainLabel} ${translations.common.and} ${totalTrainScheduleCount} ${trainScheduleLabel}`;
+    await expect(this.timetableTotalItemLabel).toHaveText(expectedComputedLabel);
+  }
+
+  async filterNameAndVerifyTrainCount(
+    name: string,
+    expectedTrainCount: number,
+    defaultValue?: string
+  ) {
+    await this.timetableFilterButton.click();
+    await this.fillTimetableLabelFilter(name);
+    await this.verifyTrainCount(expectedTrainCount);
+    if (defaultValue !== undefined) await this.fillTimetableLabelFilter(defaultValue);
+    await this.timetableFilterButtonClose.click();
+  }
```

Comment:
```suggestion
  async filterNameAndVerifyTrainCount(
    name: string,
    expectedTrainCount: number
  ) {
    await this.timetableFilterButton.click();
    await this.timetableLabelFilterInput.fill(name);
    await this.verifyTrainCount(expectedTrainCount);
    await this.timetableLabelFilterInput.clear();
    await this.timetableFilterButtonClose.click();
  }
```
and remove fillTimetableLabelFilter() method

Reply:
Done